Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.py

```diff
@@ -204,6 +230,37 @@ def validate_basic_authentication(headers: dict, username: str, password: str) -
 ''' ROUTE FUNCTIONS '''
 
 
+def get_request_args(params):
+    limit = request.args.get('n', None)
```

Comment:
```suggestion
    limit = try_parse_integer(request.args.get('n', params.get('list_size')), CTX_LIMIT_ERR_MSG)
```
Same goes for the rest, but what happens if the number is not present anywhere?